setup.py

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env python
+from setuptools import find_packages, setup
+
+setup(name='picli-gman',
+      use_scm_version=True,
+      description='A sinister inter-dimensional bureaucrat that monitors'
+                  ' the state of each FaaS.',
+      classifiers=['Development Status :: 2 - Pre-Alpha',
+                   'Environment :: Console',
+                   'Operating System :: OS Independent',
+                   'Programming Language :: Python'],
+      author='Nick Shobe',
+      author_email='nickshobe@gmail.com',
+      license='MIT License',
+      packages=find_packages(),
+      include_package_data=True,
+      zip_safe=False,
+      requires=['flask-restful',

```

Comment:
I believe this needs to be "install_requires". distutils will freak out about the '-' in the package name